app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/PlanWizardInstancePropertiesStepReducer.js

```diff
@@ -72,7 +72,11 @@ export default (state = initialState, action) => {
         } else {
           bestFitFlavor =
             tenantFlavors &&
-            tenantFlavors.reduce((prev, current) => (prev.root_disk_size > current.root_disk_size ? prev : current));
+            tenantFlavors.reduce((prev, current) => {
+              if (prev.memory > current.memory) return prev;
+              if (prev.memory < current.memory) return current;
+              return prev.cpus > current.cpus ? prev : current;
```

Comment:
We can revisit (and remove) this if/when the backend gives us this info.

But for now, this looks good!